crates/view/src/planner.rs

```diff
@@ -543,7 +543,9 @@ impl<R: RngCore + CryptoRng> Planner<R> {
         // Compute an initial fee estimate based on the actions we have so far.
         self.action_list.refresh_fee_and_change(
             &mut self.rng,
-            &self.gas_prices,
+            &self
+                .gas_prices
+                .context("planner instances must call set_gas_prices prior to planning")?,
```

Comment:
While I'm not a huge fan of forcing a separate call rather than doing it automatically, I _love_ how descriptive this error message is!

Reply:
We alternatively could just change the `Planner` constructor to require `GasPrices` or a `ViewClient` instance it can fetch them from